Source/WebCore/css/CSSAnchorValue.h

```diff
@@ -40,7 +40,8 @@ class CSSAnchorValue final : public CSSValue {
 
     void collectComputedStyleDependencies(ComputedStyleDependencies&) const;
 
-    String customCSSText() const;
+    void customCSSText(StringBuilder&) const;
```

Comment:
I think that we should add the word _append_ to these function names when changing them to append to a `StringBuilder` rather than using overloading. Unless there is a reason overloading has a benefit.